Avenged Sevenfold - Bat Country

WowItsOnlyConnor opened the pull request:
## Chart Verification PR

<!-- Here you should change these to the artist and song names. -->

Avenged Sevenfold - Bat Country

<!-- If possible, upload the album art below. -->
![album](https://github.com/user-attachments/assets/95db5a3e-9902-4b14-a3ae-421e05a173b0)

### Instruments and difficulties charted

<!-- Denote all difficulties and charts you have charted, so it's easier 
for the verifiers to verify the chart.
You can do this by changing "No" to "Yes", if you charted it. -->

| Instrument     | Easy     | Medium    | Hard      | Expert       |
| -------------- | -------- | --------- | --------- | ------------ |
| Drums          | No       | No        | No        | Yes           |
| Bass           | No       | No        | No        | Yes           |
| Lead           | No       | No        | No        | Yes           |
| Vocals         | No       | No        | No        | Yes           |
| Pro Drums      | Yes       | Yes        | Yes        | Yes/Expert+           |
| Pro Bass       | Yes       | Yes        | Yes        | Yes           |
| Pro Lead       | Yes       | Yes        | Yes        | Yes           |
| Pro Vocals       | Yes       | Yes        | Yes        | Yes           |

### Additional details

<!-- If there are any details about your chart, like for example:
intentionally placed invalid chords, no lifts or something else,
you can specify below to make the verifier aware of it. -->
Sourced from Rock Band 2 DLC.

LlysiX commented:
very good pad charts overall, lead has some lifts with note gaps that are too long to comfortably be hit as lifts
i think drums would benefit if lifts were used more, especially during rolls (not necessarily an issue, just personal preference)
bass and vocals are excellent

WowItsOnlyConnor commented:
[Removed That One Lift Note seen in that particular pattern](https://github.com/Encore-Developers/songs/pull/44/commits/53c14cd1e37146d63e0b2b1f4bd8f548d060abcf)

WowItsOnlyConnor commented:
Minor adjustments-
- Lead is now rated 6
- Vocals is now rated 4
- Removed more lifts from Lead
- Added a lift in a three gem jack patterns on Drums
- The intro Drum pattern is now in line with the other Drum patterns

WowItsOnlyConnor commented:
Trills into chords now have lift notes at the end of the trills

this should be the final commit if there's no other issues with the chart
